clients/admin-ui/src/features/privacy-experience/ConfigurePrivacyExperience.tsx

```diff
@@ -136,13 +138,36 @@ const ConfigurePrivacyExperience = ({
     TranslationWithLanguageName | undefined
   >(undefined);
 
-  const handleNewTranslationSelected = (translation: ExperienceTranslation) => {
+  const [translationIsOOB, setTranslationIsOOB] = useState<boolean>(false);
```

Comment:
Can you put the `translationIsOOB` into the same state for `setTranslateToEdit(...)` to me...? This feels like tricky state to manage independently.

Reply:
@NevilleS That was the original approach that I tried taking, where I modified the type of `translationToEdit` to bundle an `isOOB` bool with the translation, but it ended up causing some headaches.  

We want to only consider an OOB translation to be an OOB translation (for the purposes of submit behavior and showing the notice) when it's being created for the first time.  So when we're setting `translationToEdit`, we need to be able to tell whether we're creating a new translation and populating with OOB values or editing a saved translation that happens to have all the same values as an OOB translation.

This means we'd need to be able to pass that `isOOB` value on creating a new translation, which is where the main problem is: `handleCreateNewTranslation` is ultimately called from the translations' ScrollableList component, which is typed to expect specifically an ExperienceTranslation and won't like extra fields.  It would have required a lot of modifications to the ScrollableList to allow it to handle other types and conditionally pass optional properties.  So it was much cleaner to implement this way. 

Maybe a better variable name would help here to communicate that this state isn't really a property of the translation itself, but rather of what the user is currently doing.  Do you think something like `isUsingOOBValues` would be better?

Reply:
yes that rename makes sense to me